WordPress/src/main/res/values/styles.xml

```diff
@@ -85,18 +85,20 @@
         <item name="android:background">@android:color/transparent</item>
     </style>
 
-    <style name="FilteredRecyclerViewFilterTextView.WordPress" parent="android:TextAppearance.Widget.TextView">
+    <style name="FilteredRecyclerViewFilterTextView">
         <item name="android:layout_marginStart">0dp</item>
         <item name="android:paddingBottom">@dimen/margin_medium</item>
         <item name="android:paddingEnd">@dimen/margin_extra_large</item>
         <item name="android:paddingStart">10dp</item>
         <item name="android:paddingTop">@dimen/margin_medium</item>
         <item name="android:textSize">@dimen/text_sz_large</item>
+        <item name="textColor">@color/white</item>
```

Comment:
```suggestion
        <item name="textColor">@android:color/white</item>
```

Can we change this to `@android:color/white`?  There is no resource `white` in `colors.xml` so using `@color/white` would be pulling from another source or library.

Reply:
ohh, good catch. I didn't realize I had enabled "suggestions from dependencies".

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/10387/commits/be3c478d4198c852b8678ab7ff3b4f7faa633fef